app/uk/gov/hmrc/cgtpropertydisposals/connectors/S3Connector.scala

```diff
@@ -35,24 +35,24 @@ import scala.concurrent.duration._
 import scala.concurrent.{ExecutionContext, Future}
 import scala.util.control.NonFatal
 
-@ImplementedBy(classOf[UpscanConnectorImpl])
-trait UpscanConnector {
+@ImplementedBy(classOf[S3ConnectorImpl])
+trait S3Connector {
   def downloadFile(upscanSuccess: UpscanSuccess): Future[Either[Error, FileAttachment]]
 }
 
 @Singleton
-class UpscanConnectorImpl @Inject() (
+class S3ConnectorImpl @Inject() (
   playHttpClient: PlayHttpClient,
   config: ServicesConfig
 )(
   implicit executionContext: ExecutionContext,
   system: ActorSystem
-) extends UpscanConnector
+) extends S3Connector
     with Logging
     with HttpErrorFunctions {
 
   implicit val mat: ActorMaterializer        = ActorMaterializer()
-  private lazy val userAgent: String         = config.getConfString("appName", "cgt-property-disposal-frontend")
+  private lazy val userAgent: String         = config.getConfString("appName", "cgt-property-disposal")
```

Comment:
shouldn't it be `disposals` with an `s`?

Reply:
yes! well spotted

Reply:
fixed